lib/trento_web/controllers/v1/host_controller.ex

```diff
@@ -51,4 +52,33 @@ defmodule TrentoWeb.V1.HostController do
       send_resp(conn, 204, "")
     end
   end
+
+  operation :select_checks,
+    summary: "Select Checks for a Host",
+    tags: ["Checks"],
+    description: "Select the Checks eligible for execution on the target infrastructure",
+    parameters: [
+      id: [
+        in: :path,
+        required: true,
+        type: %OpenApiSpex.Schema{type: :string, format: :uuid}
+      ]
+    ],
+    request_body: {"Checks Selection", "application/json", Schema.Checks.ChecksSelectionRequest},
+    responses: [
+      accepted: "The Selection has been successfully collected",
+      not_found: Schema.NotFound.response(),
+      bad_request: Schema.BadRequest.response(),
```

Comment:
I have a doubt on this. I am tempted to remove it. I couldn't make it respond a 400.

Reply:
I took advantage by cleaning up also other controllers defining an unnecessary bad_request :see_no_evil: 

> I guess the command dispatch command returns {:error, :host_not_registered}, right?
>
yes if the host is not registered yet. It might also return `{:error, :host_rolling_up}` if the aggregate is in the rollup process.